diff --git a/src/Makefile b/src/Makefile
index 483af9b..0f1d69d 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -1,8 +1,8 @@
 export CC = g++
 
-export CCFLAGS = -g -std=c++0x -Wall
+export CCFLAGS = -g -std=c++0x -Wall $(shell python3.7-config --cflags)
 
-export LDFLAGS = -Lcore -L/usr/lib/python3.5/config-3.5m-x86_64-linux-gnu/ -lpython3.5 -lstdc++ -lpthread -lm
+export LDFLAGS = -Lcore $(shell python3.7-config --ldflags) -lstdc++ -lpthread -lm
 
 LIB_SOURCES = $(wildcard core/*.cpp)
 LIB_SOURCES += $(wildcard pcc/monitor_interval/*.cpp)
diff --git a/src/pcc/rate_control/pcc_python_rc.h b/src/pcc/rate_control/pcc_python_rc.h
index 283ba6b..ecb3f04 100644
--- a/src/pcc/rate_control/pcc_python_rc.h
+++ b/src/pcc/rate_control/pcc_python_rc.h
@@ -7,7 +7,7 @@
 #include <vector>
 
 #include "pcc_rc.h"
-#include <python3.5/Python.h>
+#include <python3.7/Python.h>
 #include <iostream>
 #include <sstream>
 
